frontend/src/routes/explore/+page.svelte

```diff
@@ -119,20 +126,36 @@
                 </div>
             </div>
 
-            <div class="level-right">
-                <div class="level-item">
-                    <span class="icon">
-                        <a href="/"><i class="fas fa-magnifying-glass" /></a>
-                    </span>
+            <div class="level-right" style="width: 60%">
+                <div class="level-item mr-3" style="width: 100%">
+                    <button class="button is-rounded" style="width: 100%">
```

Comment:
Is this button intended to be clickable?

Reply:
Yep the entire search bar is actually just one button